src/App.js

```diff
@@ -88,6 +88,10 @@ class App extends Component {
       },
       distrustConfirmed: false,
     };
+
+
+
+    console.log(getTextForLanguage(`zh`));
```

Comment:
I've done this a million times myself

Reply:
Ah, that was leftover from our call this morning.  Sorry about that.  Good catch.  Thanks!